gems/aws-sigv4/sig/error.rbs

```diff
@@ -0,0 +1,13 @@
+module Aws
+  module Sigv4
+    module Errors
+      class MissingCredentialsError < ArgumentError
+        def initialize: (?String? msg) -> void
+      end
+
+      class MissingRegionError < ArgumentError
+        def initialize: (*untyped args) -> void
```

Comment:
The args are actually not used at all... we should potentially change the code to be just `def initialize(*)`

Reply:
Fixed!
5c6600f4cf845af9eedd8ebee14c1c8175dcd544